### PR TITLE
Add op impl checkt tests

### PR DESCRIPTION
**Pull request by davidsnam-intel**
### Details:
 - *Equal-1*
 - *Greater-1*
 - *GreaterEqual-1*
 - *Less-1*
 - *LessEqual-1*
 - *NotEqual-1*
 - *LogicalAnd-1*
 - *LogicalOr-1*
 - *LogicalXor-1*
 - *Xor-1*
 - *Broadcast-1, Broadcast-3*
 - *NV12toBGR-8*
 - *NV12toRGB-8*
 - *I420toBGR-8*
 - *I420toRGB-8*
 - *If-8*
 - *MulticlassNms-8*
 - *MatrixNms-8*
 - *Assign-3, Assign-6*
 - *ReadValue-3, ReadValue-6*
 - *Elu-1*
 - *EmbeddingSegmentsSum-6*
 - *ExperimentalDetectronDetectionOutput-6*
 - *ExperimentalDectectronGenerateProposalsSingleImage-6*
 - *ExperimentalDetectronPriorGridGenerator-6*
 - *ExperimentalDetectronROIFeatureExtractor-6*
 - *ExperimentalDetectronTopKROIs-6*
 - *ExtractImagePatches-3*
 - *FakeQuantize-1*
 - *GRN-1*
 - *GRUSequence-5*
 - *GatherElements-6*
 - *GatherTree-1*
 - *Gelu-2*
 - *GroupConvolution-1*
 - *GroupConvolutionBackpropData-1*
 - *HardSigmoid-1*
 - *Interpolate-1, Interpolate-4*
 - *LRN-1*
 - *LSTMSequence-1*
 - *LogSoftmax-5*
 - *LogicalNot-1*
 - *Loop-5*
 - *MVN-2, MVN-6*
 - *MatMul-1*
 - *Mish-4*
 - *Mod-1*
 - *NonMaxSuppression-1, NonMaxSuppression-3, NonMaxSuppression-4, NonMaxSuppression-5*
 - *NonZero-3*
 - *NormalizeL2-1*
 - *OneHot-1*
 - *PRelu-1*
 - *PSROIPooling-1*
 - *Pad-1*
 - *Parameter-1*

### Tickets:
 - *78372*
 - *78395*
 - *78396*
 - *78413*
 - *78414*
 - *78440*
 - *78417*
 - *78419*
 - *78420*
 - *78995*
 - *78343, 78344*
 - *78431*
 - *78432*
 - *78402*
 - *78403*
 - *78405*
 - *78430*
 - *78425*
 - *78334, 78335*
 - *78458, 78459*
 - *78368*
 - *78371*
 - *78375*
 - *78376*
 - *78377*
 - *78378*
 - *78379*
 - *78380*
 - *78381*
 - *78383*
 - *78385*
 - *78389*
 - *78392*
 - *78393*
 - *78397*
 - *78398*
 - *78401*
 - *78406, 78407*
 - *78408*
 - *78411*
 - *78416*
 - *78418*
 - *78421*
 - *78422, 78423*
 - *78424*
 - *78428*
 - *78429*
 - *78434, 78435, 78436, 78437*
 - *78438*
 - *78439*
 - *78441*
 - *78442*
 - *78443*
 - *78444*
 - *78445*

### CODE REVIEW

File: src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp [hunk 1]
```diff
@@ -36,21 +557,201 @@ std::shared_ptr<ov::Model> generateBinaryEltwise(const std::shared_ptr<ov::op::O
...
+    ov::ResultVector results{std::make_shared<ov::op::v0::Result>(eltwise)};
+    return std::make_shared<ov::Model>(results, ngraph::ParameterVector{params}, "BinaryEltwiseLogicalGraph");
+}
+
+std::shared_ptr<ov::Model> generateBroadcast(const std::shared_ptr<ov::op::Op>& node) {
+    const ov::Shape input_shape{};
+    const ov::Shape output_shape{5, 4, 3, 2};
+    const auto params = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, input_shape);
+    const auto shape_const =
+        ov::op::v0::Constant::create(ov::element::u64, ov::Shape{output_shape.size()}, output_shape);
```
Comment (wilson-seok) [inline src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp:617]: Let's use single way to create constant node in the PR - builder::makeConstant().

File: src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp [hunk 2]
```diff
@@ -10,13 +10,534 @@ namespace test {
 namespace subgraph {
 
 namespace {
-std::shared_ptr<ov::Model> generate(const std::shared_ptr<ov::op::Op> &node) {
+std::shared_ptr<ov::Model> generate(const std::shared_ptr<ov::op::Op>& node) {
```
[the remark this answers is not in the document]
Comment (davidsnam-intel) [reply src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp:13]: No special reason to use like it. For consistency with others, I'll change it all to `&node`.

File: src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp (hunk 1 above)
Comment (davidsnam-intel) [reply src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp:617]: I'll update by applying `ngraph::builder::makeConstant()`

**Review by wilson-seok — APPROVED** [review]

**Review by steve-y — APPROVED** [review]

**Comment by iefode** [comment]
Report: https://openvino-ci.intel.com/job/private-ci/job/nightlyJobs/job/e2e-tests-linux/22631/gen_5fconformance_5ftests_5fsummary/

**Review by iefode — APPROVED** [review]

**Pull request merged**